clap-utils/src/keypair.rs

```diff
@@ -170,6 +170,19 @@ pub(crate) enum SignerSourceKind {
     Pubkey(Pubkey),
 }
 
+impl std::fmt::Debug for SignerSourceKind {
+    fn fmt(&self, fmt: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let source = match self {
+            SignerSourceKind::Prompt => "prompt",
+            SignerSourceKind::Filepath(_) => "filepath",
+            SignerSourceKind::Usb(_) => "usb",
+            SignerSourceKind::Stdin => "stdin",
+            SignerSourceKind::Pubkey(_) => "pubkey",
+        };
```

Comment:
wdyt about implementing the `match` statement in `AsRef<str>` and calling that here?

Reply:
Done in https://github.com/solana-labs/solana/pull/17180/commits/abdcf3da9134a3e67e378e78fd8545955390ff9a
Any objections to using consts for the URI scheme match?

Reply:
I will never object to using `const`s :wink: